espanya/104_viseloga.md

```diff
@@ -0,0 +1,159 @@
+
+## Pronombres personales
+
+| Singular    | Plural       |
+|:------------|:-------------|
+| **mi**      | **mimon**    |
+| yo          | nosotros     |
+| **tu**      | **tumon**    |
+| tú, usted   | ustedes      |
+| **ye**      | **yemon**    |
+| él, ella    | ellos, ellas |
+
+All pronouns can be used for all genders.
+
+The possessive pronouns consists of the personal pronoun and the possessive particle
+**su**.
+
+| Singular    | Plural       |
+|:------------|:-------------|
+| **mi su**   | **mimon su** |
+| mi          | nuestro      |
+| **tu su**   | **tumon su** |
+| tu          | su           |
+| **ye su**   | **yemon su** |
+| su          | su           |
+
+
+## Pronombre reflexivo
+
+El pronombre reflexivo se usa cuando el objeto de una oración es la misma como el sujeto.
+
+**se**
+– se  
+**mi vide se.**
+– Se veo.  
+**mimon vide se.**
+– Nos vemos.
+
+La expresión **se kon** se usa cuando dos o más personas performan la acción el uno al otro.
+
+**se kon**
+– el uno al otro
+
+**mi e tu vide se kon.**
+– Yo y tú vemos el uno al otros.  
+**mimon vide se kon.**
+– Se vemos.
+
+
+## Pronombres demostrativos
+
+Pronombres demostrativos se usan con sustantivos para hacerlos más espicificos.
+Los demostrativos en Pandunia son:
+
+**ni**
+– esto (cerca del orador)  
+**go**
+– eso (lejos del orador)  
+**la**
+– él o la (conocido por el orador y el oyente)
+
+En Pandunia, **ni**, **go**, y **la** funciona solo como modificadores,
+y requieren un sustantivo despés.
+Para usarlos como pronombres, un sustantivo genérico como
+**she**
+(«cosa») se necesita poner después.
+
+El demostrativo proximal
+**ni**
+indica cosas que están cerca del orador.
+El demostrativo distal
+**go**
+indica cosas que están lejos del orador.
+
+**ni she si bon.**
+– Esta (cosa) es buena.  
+**go she si dus.**
+– Esa (cosa) es mala.  
+**mi vol ni buku, no go.**
+– Quiero este libro, no ese.
+
+Los demostrativos proximal y distal se usan para introducir objetos nuevos.
+El demostrativo temático, en el otro lado,
+no espicifica distancia fisical,
+pero se usan cuando el orador ya mencionó el objeto o persona en cuestión,
+o es conocido por el oyente o es de interés actual en el discurso.
+
+**ni she si mau. ye vol yam go mushu.**
+– Ese es gato.  Quiere comer ese ratón.
+
+**mi ten un mau e un vaf. la vaf si dai. ye yam poli yam.**
+– Tengo un gato y un perro. El perro es gran. Come mucha comida.
+
+
+### Uso abstracto
+
+Los demostrativos se pueden usar para referir
+a entidades abstractas de discurso, no objetos concretos.
+**la** refiere a cosas que se dijeron anteriormente,
+**ni** refiere a cosas que se dicen actualmente,
+y **go** refiere a cosas que se dirán en el futuro.
+
+**ni jumle si korte.**
+– Esta oración es corta.
+
+Arriba, **ni jumle** (_esta oración_) refiere a la oración que se está hablando.
+
+**mi mana go she: mi ama tu.**
+– Significo esta: te amo. (Significo que te amo.)  
+**mi ama tu. mi mana la she.**
```

Comment:
Using ni, go and la this way makes sense. Let's update the English version accordingly.